docroot/themes/custom/twentyseventeen/sass/base/form/_form.scss

```diff
@@ -201,3 +229,139 @@ input[type="time"][disabled],
 input[type="url"][disabled] {
   background-color: $lt-gr;
 }
+// Select Button Styling
+select {
+  -webkit-appearance: none;  /*Removes default chrome and safari style*/
```

Comment:
I think this should be `appearance: none` for autoprefixr from skimming this https://github.com/postcss/autoprefixer/issues/43, but we should check.